stylix/overlays.nix

```
@@ -0,0 +1,20 @@
+{
+  lib,
+  inputs,
+  config,
+  ...
+}@args:
+(lib.singleton {
+  options.stylix.overlays.enable = config.lib.stylix.mkEnableTarget "Enable theming of some packages via overlays" true;
```

`config.lib.stylix.mkEnableTarget` generates the following:

```nix
description: "Whether to enable theming for ${description}."
```

With that in mind, the description should be changed to something like:

```suggestion
  options.stylix.overlays.enable = config.lib.stylix.mkEnableTarget "packages via overlays" true;
```